package.json

```diff
@@ -24,9 +24,8 @@
     "stylelint": "stylelint packages/commonwealth/client/styles/*",
     "chromatic": "chromatic --exit-zero-on-changes",
     "prepare": "husky install & yarn workspace @hicommonwealth/core build",
-    "watch-core": "tsc --watch --project libs/core/tsconfig.json",
-    "dev-server": "yarn workspace commonwealth start",
-    "dev": "concurrently -k -c 'auto' -n 'core,server' 'yarn watch-core' 'yarn dev-server'"
+    "dev": "nodemon --watch './libs/core/src/**' --ext 'ts,tsx' --ignore './libs/**/*.spec.ts' --exec 'yarn workspace @hicommonwealth/core build && yarn workspace commonwealth start'",
```

Comment:
Doesn't adding `tsconfig-paths` mean we don't need to compile and use `@hicommonwealth/core` as a package during development since `ts-node` or `ts-node-dev` can just resolve to the libs dir directly?

Reply:
Additionally, if we must use `nodemon`, this command doesn't cover other servers like the commonealth-consumer, EVM chain-events, or the cosmos notification generator. Do we create separate dev commands for each of those?

Reply:
yes, I need your help with all active services (not sure how many we are deprecating)... we can use the same patterns - anything using ts-node/ts-node-dev should register the paths with tsconfig-paths... and we have to nodemon if we want hot reloads in dev mode.

Reply:
it only allows ts-nodish to find the paths configured in tsconfig, but the node resolution stills finds the libs in node_modules

Reply:
I don't think this conversation is resolved. If you run `yarn start` directly from the commonwealth package and then modify one of the lib files like `platform.ts` hot-reload already works out of the box. I don't see why we need 2 layers of hot reloading: one with nodemon to build and then another one nested inside with `ts-node-dev` for the commonwealth package.